roles/build-release/tasks/setup.yaml

```diff
@@ -1,15 +1,4 @@
 ---
-- name: Install antsibull from current source in a virtualenv with poetry
-  command: poetry install
-  args:
-    chdir: "{{ playbook_dir | dirname }}"
-
-# Existing poetry installs are not updated and it could lead to stale deps
-- name: Ensure poetry virtualenv is up to date
-  command: poetry update
-  args:
-    chdir: "{{ playbook_dir | dirname }}"
```

Comment:
I see where you are going but I'm a bit torn on this -- on one hand, antsibull is assumed to be installed but the role was previously taking care of that.

Should the role create a virtualenv in which it'd install antsibull (from source or from pypi) instead ?

Reply:
I think it's better to make a role as agnostic as possible on the tools used. The old version was basically forcing everyone to use antsibull in a very specific way (namely using poetry). The current version allows users to install antsibull howevery they like - in a venv, `sudo pip install antsibull`, run it with poetry, even install it as a system package (if anyone wants to package it). I don't see why we should once again force a very specific way to run antsibull.

What we could do is provide another role that installs antsibull in a specific way (using venv for example), so that if both roles are used together you get the desired result - but you aren't forced to do that. But that really shouldn't be part of a role whose job is to build Ansible.

Reply:
> The old version was basically forcing everyone to use antsibull in a very specific way (namely using poetry)

Indeed, the original intent was to be a role written based on the existing documented way of building ansible (https://github.com/ansible-community/antsibull/blob/main/docs/build-ansible.rst). In other words, automating these manual steps and providing test coverage where there was none before.

> The current version allows users to install antsibull howevery they like - in a venv, sudo pip install antsibull, run it with poetry, even install it as a system package (if anyone wants to package it). I don't see why we should once again force a very specific way to run antsibull.

Fine with me.

> What we could do is provide another role that installs antsibull in a specific way (using venv for example), so that if both roles are used together you get the desired result - but you aren't forced to do that. But that really shouldn't be part of a role whose job is to build Ansible.

:+1: